aws-cpp-sdk-cognitoidentity-integration-tests/IdentityPoolOperationTest.cpp

```diff
@@ -129,7 +130,7 @@ class IdentityPoolOperationTest : public ::testing::Test
             deleteIdentityPoolRequest.WithIdentityPoolId(pool.GetIdentityPoolId());
 
             DeleteIdentityPoolOutcome deleteIdentityPoolOutcome = client->DeleteIdentityPool(deleteIdentityPoolRequest);
-            ASSERT_TRUE(deleteIdentityPoolOutcome.IsSuccess());
+            ASSERT_TRUE(deleteIdentityPoolOutcome.IsSuccess()) << deleteIdentityPoolOutcome.GetError().GetMessage();
```

Comment:
I will replace with a macro in a follow-up PR for the rest of integration test.
(this PR has passed the CI build and I don't want to update this PR at this moment).